### DISCUSSION TITLE
Paper to cite for MatPlot++

### DISCUSSION THREAD

**Discussion by pbenharris** [alandefreitas/matplotplusplus#359] [General]
First of all, let me echo what others have stated: thank you for creating and sharing this library! I am creating a massive N-body simulation that must be performed in C++ and this library may be the best way to visualize the results. The simulation is for a project that predicts how Earth accumulates dark matter. I can 100% see using the alpha blended scatter plots and heatmaps being necessary for this work. 

I plan to write several journal publications for this project. I would like credit MatPlot++ in the Acknowledgements and References. **Is there a journal article I can cite for MatPlot++?** I don't mind citing the website, but if anyone has published a journal article on this project, I wouldn't want to miss that. I couldn't find anything on Google scholar, but, then again, the hits all come back about Matplotlib.

Best regards,
Ben

**Comment by alandefreitas** [comment]
Thanks, @pbenharris. I hope the library is useful. I started writing a paper at the time but never got to finish it. At this point, citing the website is the best we can do. That actually has its pros, since that's where users will get the most useful information instead of some arbitrary comparison with data visualization techniques. 

I guess citing the paper is only beneficial in terms of boosting impact indicators, which I couldn't care less about at this point. As long as the library is useful to as many people as possible, our goal has been achieved. :)
